api-report/datastore-definitions.api.md

```diff
@@ -120,7 +120,7 @@ export interface IFluidDataStoreRuntime extends IFluidRouter, IEventProvider<IFl
     // (undocumented)
     readonly objectsRoutingContext: IFluidHandleContext;
     // (undocumented)
-    readonly options: ILoaderOptions;
+    options: ILoaderOptions;
```

Comment:
I'm not convinced that removing the readonly constraint is the correct thing to do here, but I wasn't able to change the options on the runtime object otherwise.